plugins/cleanowned.cpp

```diff
@@ -160,7 +169,7 @@ command_result df_cleanowned (color_ostream &out, vector <string> & parameters)
             {
                 if (!Items::setOwner(item,NULL))
                     out.print("(unsuccessfully) ");
-                if (dump)
+                if (dump && !nodump)
```

Comment:
would it be possible to not set `dump=true` in the first place instead of canceling it out here? that might make the semantics clearer in the code.

Reply:
I can move the `dump = true` into the else statements one second

Reply:
Moved for better readability